tower-http/src/trace/on_response.rs

```diff
@@ -200,7 +204,7 @@ macro_rules! log_pattern_match {
 }
 
 impl<B> OnResponse<B> for DefaultOnResponse {
-    fn on_response(self, response: &Response<B>, latency: Duration) {
+    fn on_response(self, response: &Response<B>, latency: Duration, _span: &Span) {
```

Comment:
slight preference for
```suggestion
    fn on_response(self, response: &Response<B>, latency: Duration, _: &Span) {
```